nvm.1

```diff
@@ -0,0 +1,152 @@
+.TH "NVM" "1" "March 2018" "NVM" "nvm"
```

Comment:
Hm, I'm not sure what you mean exactly. I'm not sure if you mean that line in particular or the entirety of the manpage. The date is expected to only be updated when the manpage is update, and can be removed entirely realistically. It was originally there for version control but that's not very useful anymore, especially since this is a git repository.

If you mean programmatically generate the entire manpage, then sure it's possible, but would require more work. I also don't know if it's desired, but that's not up to me.

Reply:
Sorry for being unclear; I'm referring to the entirety of the manpage.

I alter the content of `nvm --help` relatively frequently (hopefully improvements) and I think that without automatic enforcement (ensuring that the manpage matches) and a script to synchronize the content (see `npm run doctoc` for an example) that it'd be too easy to drift.

In summary, if you're willing to do the work to automatically generate the entire manpage and provide a reproducible script for that, that'd be awesome.

Reply:
I might, but I'd like to think about this a bit more. In my opinion, user-facing documents shouldn't be autogenerated. I could just do a pretty straightforward translation of the `nvm --help` output, but I think something is lost in that. Alternatively, something could just check to see if any commit changes the output of `nvm --help`, and if so, not allow a merge without updating the manpage. That seems a little bit aggressive, but also keeps things more in check (hopefully).

If you have any other suggestions, let me know.